tox.ini

```
@@ -82,7 +82,8 @@ envlist =
     mongodb-datastore_pymongo-{py27,py36,py37,py38,py39,py310,pypy}-pymongo{03},
     mysql-datastore_pymysql-{py27,py36,py37,py38,py39,py310,pypy,pypy3},
     solr-datastore_pysolr-{py27,py36,py37,py38,py39,py310,pypy,pypy3},
-    redis-datastore_redis-{py27,py36,py37,py38,py39,py310,pypy,pypy3},
+    redis-datastore_redis-{py27,py36,py37,py38,pypy,pypy3}-redis03,
+    redis-datastore_redis-{py36,py37,py38,py39,py310,pypy3}-redis{040,latest},
```

Let's keep 2 digit number for all versions so this doesn't get confusing.

```suggestion
    redis-datastore_redis-{py36,py37,py38,py39,py310,pypy3}-redis{0400,latest},
```